crates/rover-client/src/operations/persisted_queries/describe_pql/describe_pql_query.graphql

```diff
@@ -0,0 +1,13 @@
+query DescribePersistedQueryListQuery($graphId: ID!, $variant: String!) {
```

Comment:
Is it worth throwing a comment in these files noting that before adding any fields to these files it's essential to ensure that the fields in question are accessible to graph keys with role `PERSISTED_QUERY_PUBLISHER`? (If you look at the PR in the internal GraphOS repo you will see that that's a pretty limited set of fields, intentionally.)